tutorials/ai-core-aiapi-postman-setup/ai-core-aiapi-postman-setup.md

```diff
@@ -24,6 +24,8 @@ Download the following files
 |  `AI-API_postman_collection.json` | [Download Here](https://raw.githubusercontent.com/SAPDocuments/Tutorials/master/tutorials/ai-core-aiapi-postman-setup/AI-API_postman_collection.json)
 |  `AI-API_postman_environment.json` | [Download Here](https://raw.githubusercontent.com/SAPDocuments/Tutorials/master/tutorials/ai-core-aiapi-postman-setup/AI-API_postman_environment.json)
 
+> Note: Look for the latest collection of APIs for SAP AI Core [here](https://api.sap.com).
```

Comment:
Please don't start a note with the label "note".

https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9?_lightbox=true#mcetoc_1dtqfqchq21